.github/workflows/test.yaml

```
@@ -76,9 +76,19 @@ jobs:
       - name: Lint OpenAPI specification file
         run: docs/openapi/openapi-lint
 
+  check-no-unwanted-strings:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Check unwanted tags
```

I suggest

```suggestion
      - name: Ensure no debug tags are committed in feature files
```

(Just personal preference, and makes the step intentions clearer.)